### ISSUE TITLE
cli: Protein Parameter swallows error messages

### ISSUE THREAD

**Issue by richardjgowers** [OpenFreeEnergy/openfe#494]
From here:

https://github.com/OpenFreeEnergy/ExampleNotebooks/issues/77

The actual error was a nonstandard residue in the PDB, this error message was swallowed by the strategy selector system in the CLI system however.

Ideally we'd have also seen the error message from the `ProteinComponent.from_pdb` call that was more informative

**Comment by dwhswenson** [comment]
Yeah, we should probably be using plain `click` types, instead of the `get` approach from `plugcli`. We'll use `get` for things like  loading a specific transformation from the new storage (this is what that functionality was originally designed for, where the options refer to objects it a database, which might be named, or might be requested by index, or might be optional if there's only one object of that type in the database.).
